src/main/java/com/github/dockerunit/annotation/WithSvc.java

```diff
@@ -25,40 +25,40 @@
  */
 @Retention(RUNTIME)
 @Target({ TYPE, METHOD })
-@Repeatable(Usages.class)
-public @interface Use {
+@Repeatable(RepeatableWithSvc.class)
+public @interface WithSvc {
 
     /**
-     * The descriptor class that defines how to instantiate the service
+     * The descriptor class that defines how to instantiate the svc
      *
      * @return the descriptor type
      */
-    Class<?> service();
+    Class<?> svc();
```

Comment:
Now that I looked at the PRs I wonder if this should be called `descriptor` or `svcDescriptor` :thinking: 

Anyway if it's too much of a chore to change it, leave it for 3.0 :wink:

Reply:
lol :-D